torch/nn/modules/container.py

```diff
@@ -161,6 +168,8 @@ def __init__(self, parameters=None):
             self += parameters
 
     def __getitem__(self, idx):
+	if idx >= len(self._parameters):
```

Comment:
To account for negative indices? This change should be made for the other classes here as well, right?